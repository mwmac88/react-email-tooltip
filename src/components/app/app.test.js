import React from 'react';
import { shallow } from 'enzyme';
import App from './index';

describe('App Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('successfully renders 3 child components', () => {
    expect(wrapper.children()).toHaveLength(3);
  });
});
