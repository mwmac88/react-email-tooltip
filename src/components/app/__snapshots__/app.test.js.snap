// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders without crashing 1`] = `
<div>
  <GlobalStyleComponent />
  <HeaderComponent
    onClickIcon={[Function]}
  />
  <EmailSubscribe
    toggleTooltip={[Function]}
    visible={false}
  />
</div>
`;
